data/json/items/fluff.json

```
@@ -97,11 +97,25 @@
     "symbol": "*",
     "color": "light_gray"
   },
+  {
+    "id": "checkpaper",
+    "type": "GENERIC",
+    "category": "other",
+    "name": { "str": "checkered paper", "str_pl": "checkered papers" },
+    "description": "A sheet of paper scrawled with a pattern identical to the checkerboard on a chessboard.  Only useful for trying a game of chess or checkers and nothing else.",
+    "weight": "3 g",
+    "volume": "6 ml",
+    "longest_side": "297 mm",
+    "material": [ "paper" ],
+    "symbol": "?",
+    "color": "white",
+    "flags": [ "NO_REPAIR" ]
+  },
   {
     "id": "chess",
     "type": "GENERIC",
     "category": "other",
-    "name": { "str": "chess set" },
+    "name": { "str": "chess set", "str_pl": "chess sets" },
```

if plural form just adds `s`, there is no need to add `srt_pl`, the game adds it automatically

```suggestion
    "name": { "str": "chess set" },
```